README.md

```diff
@@ -76,29 +76,16 @@ restart as well.
 
 You can pull our Docker images from [Docker Hub](https://hub.docker.com/r/timescale/timescaledb/).
 
-```bash
-docker pull timescale/timescaledb:latest
-```
+To run, simply issue the following command:
 
-To run, you'll need to specify a directory where data should be
-stored/mounted from on the host machine. For example, if you want
-to store the data in `/your/data/dir` on the host machine:
 ```bash
-docker run -d \
-  --name timescaledb \
-  -v /your/data/dir:/var/lib/postgresql/data \
-  -p 5432:5432 \
```

Comment:
I would prefer to not have the -v in the default start command but would be happy to have a note saying that you have the option to add it.